main.go

```diff
@@ -1,16 +1,21 @@
 package main
 
 import (
+	"fmt"
 	"os"
+
+	"github.com/fatih/color"
 )
 
 func main() {
-
-	if len(os.Args) == 1 {
+	cli := parseArgs(os.Args)
```

Comment:
Perhaps `options` is a better name. `cli` suggests something more and less than these options.

Reply:
I can find examples of options structs in the wild that look much like this struct if you like, or need persuasion.

Reply:
Thanks. Do you mean only var name or type name also?

Reply:
Both. I meant to comment the struct too.

Not a material difference, i know.